fractal_tasks_core/lib_tables/__init__.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2022 (C) Friedrich Miescher Institute for Biomedical Research and
+# University of Zurich
+#
+# Original authors:
+# Tommaso Comparin <tommaso.comparin@exact-lab.it>
+#
+# This file is part of Fractal and was originally developed by eXact lab S.r.l.
+# <exact-lab.it> under contract with Liberali Lab from the Friedrich Miescher
+# Institute for Biomedical Research and Pelkmans Lab from the University of
+# Zurich.
+"""
+Functions and classes related to table specifications (see
+https://fractal-analytics-platform.github.io/fractal-tasks-core/tables)."""
+from typing import Any
+from typing import Optional
+
+import anndata as ad
+import zarr.hierarchy
+
+from .v1 import _write_table_v1
+from fractal_tasks_core import __FRACTAL_TABLE_VERSION__
+
+
+def write_table(
+    image_group: zarr.hierarchy.Group,
+    table_name: str,
+    table: ad.AnnData,
```

Comment:
Should we be requiring AnnData tables in this (potentially generic) API?
Our V1 writer needs AnnData tables, future versions may accept other types here